### PR TITLE
fs walk: adding backward compat updates to bwlimit parsing - fixes issue #7595

### PR DESCRIPTION
**Pull request by KDreynolds**
#### What is the purpose of this change?

The problem is the conflict between rclone's bwlimit timetable syntax, which uses spaces as separators, and Docker's formatting. This leads to frequent parsing issues when configuring rclone within Docker environments.

#### Was the change discussed in an issue or in the forum before?

#7595 

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Review by ncw — COMMENTED** [review]
What is the purpose of this? (I can see what it does - why do we need it :-)

It needs test updates and doc updates if we are to merge it!

**Comment by KDreynolds** [comment]
Its possible I misunderstood the request in issue #7595, ill amend the commit with test updates and doc updates, thanks!

**Comment by ncw** [comment]
Sorry I missed the link to the original commit which is perfectly clear :-)

Please carry on and add some docs / tests and I will merge - thank you :-)

**Comment by KDreynolds** [comment]
Ok I have made those additions. Please let me know if anything needs to be addressed @ncw. Thanks!

**Review by ncw — COMMENTED** [review]
Code and tests great - thank you :-)

This needs a note in [the docs](https://rclone.org/docs/#bwlimit-bandwidth-spec)  the source for which is in [docs/content/docs.md](https://github.com/rclone/rclone/blob/master/docs/content/docs.md)

Thank you

**Comment by KDreynolds** [comment]
Updates made, I think should be good to go now. Thanks again!

File: docs/content/commands/rclone.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ rclone [flags]
       --box-token-url string                                Token server url
       --box-upload-cutoff SizeSuffix                        Cutoff for switching to multipart upload (>= 50 MiB) (default 50Mi)
       --buffer-size SizeSuffix                              In memory buffer size when reading files for each --transfer (default 16Mi)
-      --bwlimit BwTimetable                                 Bandwidth limit in KiB/s, or use suffix B|K|M|G|T|P or a full timetable
+      --bwlimit BwTimetable                                 Bandwidth limit in KiB/s, or use suffix B|K|M|G|T|P or a full timetable. Entries can be separated by spaces or semicolons.
```
Comment (ncw) [inline docs/content/commands/rclone.md:126]: Can you drop this change - `docs/content/commands/rclone.md` is autogenerated (it says at the top) and I don't think we need the command line help getting any longer.

**Review by ncw — COMMENTED** [review]
Docs good, but one change we need to remove - see inline.

Thank you :-)

File: docs/content/commands/rclone.md (hunk 1 above)
Comment (KDreynolds) [reply docs/content/commands/rclone.md]: Oops, that's my mistake. fixed now.

**Comment by KDreynolds** [comment]
Good now I believe.

**Review by ncw — APPROVED** [review]
Thank you :-)

I'll merge now.

**Pull request merged**